Makefile

```diff
@@ -1,6 +1,9 @@
 # Simple makefile to simplify repetitive build env management tasks under posix
 
-CODESPELL_DIRS ?= ./
+# Directories to run style checks against
+CODE_DIRS ?= "doc examples examples_flask pyvista tests"
+# All files in toplevel directory
+CODE_FILES ?= $(shell find . -maxdepth 1 -type f)
```

Comment:
Is the `find` call actually better here?
1. Is this line portable? (I'm thinking windows...)
2. Don't we want to specify extensions? Things like `.gitignore` aren't code files.
3. Why not just the explicit `*.py *.md` etc globs?

Reply:
The upside is that any new files that are added would be automatically covered with find.

> Is this line portable? (I'm thinking windows...)

For this reason alone, I changed this.  I can't test it.

> Don't we want to specify extensions? Things like .gitignore aren't code files.

It is somewhat nice to check codespell for these files.  It could catch typo like errors in configuration, but it is less important.